webpack.playground.config.js

```diff
@@ -0,0 +1,72 @@
+/* global module __dirname */
+
+const path = require('path');
+const fs = require('fs');
+
+const TARGET = path.resolve(__dirname, 'target');
+const PLAYGROUND = path.resolve(__dirname, 'playground');
+const DEFAULT_SERVER_PORT = 3000;
+const WIDGET_RC = '.widgetrc';
+
+let widgetRc = {
+  widgetOptions: {
+    baseUrl: 'https://your-org.okta.com',
```

Comment:
The placeholder text we use for all open source repos is `https://{yourOktaDomain}`, as it can imply a vanity URL, preview orgs and prod orgs.

Can we change this value to be consistent with our API docs + other repos?